### PR TITLE
Update docker documentation

### PR DESCRIPTION
**Pull request by JTeijema**
This PR will update the docker documentation. The main addition is the connection between a container and a local project folder.

The readme was also updated.

### CODE REVIEW

**Comment by JTeijema** [comment]
See https://github.com/JTeijema/asreview/tree/docker_storage_fix/docker for the updated readme file.

**Comment by JTeijema** [comment]
image

Using the information available in the readme, it should be possible to create these containers. If you are missing steps for this result, please let me know in this PR, and I will expand the readme.

File: docker/asreview-cli/Dockerfile [hunk 1]
```diff
@@ -1,3 +1,4 @@
 FROM python:3.8
-RUN pip install asreview asreview-covid19 asreview-statistics asreview-visualization
+RUN pip install asreview asreview-statistics asreview-visualization
 ENTRYPOINT ["asreview"]
+ENV ASREVIEW_PATH=project_folder
```
Comment (terrymyc) [inline docker/asreview-cli/Dockerfile:4]:
```suggestion
ENV ASREVIEW_PATH=project_folder

```

File: docker/asreview-lab/Dockerfile [hunk 2]
```diff
@@ -1,3 +1,4 @@
 FROM python:3.8
-RUN pip install asreview asreview-covid19
+RUN pip install asreview
 ENTRYPOINT ["asreview","lab","--ip","0.0.0.0"]
+ENV ASREVIEW_PATH=project_folder
```
Comment (terrymyc) [inline docker/asreview-lab/Dockerfile:4]:
```suggestion
ENV ASREVIEW_PATH=project_folder

```

File: docker/asreview-cli/Dockerfile [hunk 3]
```diff
@@ -1,3 +1,4 @@
 FROM python:3.8
-RUN pip install asreview asreview-covid19 asreview-statistics asreview-visualization
+RUN pip install asreview asreview-statistics asreview-visualization
```
Comment (J535D165) [inline docker/asreview-cli/Dockerfile:2]: Why is asreview-covid19 removed?

File: docker/README.md [hunk 4]
```diff
@@ -1,43 +1,95 @@
...
+
+To attach the container to your local project folder (or any other local
+folder), the -v flag can be used in the following way. Adjust
+`path-to-your-folder` to your local folder. When a project folder is specified,
+ASReview LAB will store and load all its projects from this folder.
+Note that multiple containers can access the same folder, making switching
+between versions even easier than before.
 
+```
+docker create --name asreview-lab -p 5000:5000 -v path-to-your-folder:/project_folder asreview/asreview
```
Comment (J535D165) [inline docker/README.md:68]: Interesting solution. I'm wondering why `/.asreview` doesn't work here, but I assume it doesn't :)

**Review by J535D165 — COMMENTED** [review]
Very nice to have this PR. Looks super useful to me.

File: docker/README.md (hunk 4 above)
Comment (JTeijema) [reply docker/README.md:68]: I tried, but it did not. I think I'll have to take a look inside a container to see what folder it does store too, but that will take a while.

File: docker/asreview-cli/Dockerfile (hunk 3 above)
Comment (J535D165) [reply docker/asreview-cli/Dockerfile:2]: Yes, support is still there and planned for v1. https://github.com/asreview/asreview-covid19/pull/41

Although we might want to reconsider this. I'm fine with archiving now that the dataset has become very big and COVID19 has less impact on daily life.

File: docker/README.md (hunk 4 above)
Comment (J535D165) [reply docker/README.md:68]: Nice solution for now :)

File: docker/asreview-cli/Dockerfile (hunk 3 above)
Comment (J535D165) [reply docker/asreview-cli/Dockerfile:2]: We decided not to update the package anymore https://github.com/asreview/asreview-covid19/pull/46

**Review by J535D165 — APPROVED** [review]

**Comment by J535D165** [comment]
Very nice PR. I'm going to use this myself.

**Pull request merged**